packages/forms-web-app/src/controllers/examination/check-submission-item/utils/get-summary-list.js

```diff
@@ -0,0 +1,42 @@
+const { getActiveSubmissionItem } = require('../../session/submission-items-session');
+const {
+	personalInformationWhichIsValid
+} = require('./summary-list-item/personal-information-which/utils/is-valid');
+const {
+	getSummaryListItemEnterComment,
+	getSummaryListItemEvidenceOrComment,
+	getSummaryListItemSelectFile,
+	getSummaryListItemPersonalInformation,
+	getSummaryListItemPersonalInformationWhich,
+	getSummaryListItemSubmissionItem
+} = require('./summary-list-item');
+
+const getSummaryList = (session) => {
+	const activeSubmissionItem = getActiveSubmissionItem(session);
+
+	const summaryListItems = [];
+
+	summaryListItems.push(getSummaryListItemSubmissionItem(activeSubmissionItem));
+	summaryListItems.push(getSummaryListItemEvidenceOrComment(activeSubmissionItem));
+
+	const summaryListItemSelectFile = getSummaryListItemSelectFile(activeSubmissionItem);
```

Comment:
could these conditions also be functions ? 

even if just tenery returns outside the function. might be more descriptive.